test/EFCore.MySql.FunctionalTests/appsettings.json

```diff
@@ -2,9 +2,9 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Error",
-      "System": "Error",
-      "Microsoft": "Error"
+      "Default": "Trace",
```

Comment:
Hello Caleb, true, should not be part of my commit, it seems that i can't unstage it myself (well, at least i don't know how to proceed in github).